Source/Features/Store/NotificationStoreCombine.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  NotificationStoreCombine.swift
+//  MagicBell
+//
+//  Created by Javi on 27/12/21.
+//
+
+import Foundation
+import Combine
+
+public extension NotificationStore {
```

Comment:
@jfilowk Missing methods for Fetch & Refresh

Also, all methods should have documentation.